src/data/eventregistration_de.json

```diff
@@ -11,8 +11,8 @@
                 "description": "Bezeichnung",
                 "address": "Ort",
                 "defaultcheckinlength": "Typische Aufenthaltsdauer (Minuten)",
-                "starttime": "Startzeit",
-                "endtime": "Endzeit",
+                "starttime": "Beginn",
```

Comment:
How about the English version? It still has the old labels
```
"starttime": "Start time",
"endtime": "End time",
```

Reply:
Good point.